impl/client/src/test/javascript/pentaho/visual/action/Message.spec.js

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright 2022 Hitachi Vantara.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+    "pentaho/visual/action/Base",
+    "pentaho/visual/action/Message",
+    "tests/pentaho/util/errorMatch"
+], function (BaseAction, MessageAction, errorMatch) {
+
```

Comment:
Formatting should use 2 spaces indentation.

Reply:
Fixed